sdk/storage/Azure.Storage.Files.DataLake/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Storage Files Data Lake client library for .NET
 
-> Server Version: 2019-02-02
+> Server Version: 2019-07-07
```

Comment:
We also need to change the preview version in the Installation section since NuGet won't install the latest for previews.

Reply:
Fixed.